src/app/posts/[slug]/page.jsx

```diff
@@ -23,23 +23,24 @@ export async function getStaticPaths() {
   return { paths, fallback: false };
 }
```

Comment:
I cannot add comment to line 7. that function is now `generateStaticParams`. [Docs](https://nextjs.org/docs/app/api-reference/functions/generate-static-params).